apps/docs/docs/api/configuration/babel-plugin.mdx

```diff
@@ -10,106 +10,112 @@ sidebar_position: 1
 
 ## Configuration options
 
-### `dev`
+### `aliases`
 
 ```ts
-dev: boolean // Default: false
+aliases: {[key: string]: string | Array<string>} // Default: undefined
 ```
 
-When `true`, StyleX will insert debug class names to identify the source of the styles.
+`aliases` option allows you to alias project directories to absolute paths, making it easier to import modules.
 
-<hr />
+Example: `'@/components/*': [path.join(__dirname, './src/components/*')]`
 
-### `test`
+---
+
+### `classNamePrefix`
 
 ```ts
-test: boolean // Default: false
+classNamePrefix: string // Default: 'x'
 ```
 
-When `true`, StyleX will only output debug classNames identifying the
-source of the styles.
+Prefix to applied to every generated className.
 
-It will *not* generate any styles or functional classNames.
-This can be useful for snapshot testing.
+---
 
-<hr />
+### `debug`
 
-### `runtimeInjection`
+```ts
+debug: boolean // Default: false
+```
+
+When `true`, StyleX will insert debug class names to identify the source of the styles.
+
+---
+
+### `dev`
 
 ```ts
-runtimeInjection: boolean // Default: the value of `dev`
+dev: boolean // Default: false
 ```
 
-Should StyleX generate code to inject styles at runtime?
-This may be useful during development but should be disabled in production.
+When `true`, StyleX will insert debug class names to identify the source of the styles.
```

Comment:
`dev` also insert function calls to inject the styles at runtime making it possible to use StyleX without setting up CSS file extraction.

Additionally, it makes hot reloading more usable.